crates/quantum_info/src/sparse_observable/mod.rs

```diff
@@ -400,12 +403,260 @@ pub struct SparseObservable {
     /// always an explicit zero (for algorithmic ease).
     boundaries: Vec<usize>,
 }
+// ---------------------------------------------------------------------------
+//  Local 2x2 operator tables for BitTerms (Paulis + projectors)
+// ---------------------------------------------------------------------------
+
+#[derive(Clone, Copy, Debug)]
+struct LocalEntry(u8, Complex64); // (output_bit, amplitude)
+
+#[derive(Clone, Copy, Debug)]
+struct Local2x2 {
+    out0: &'static [LocalEntry], // transitions when input bit = 0
+    out1: &'static [LocalEntry], // transitions when input bit = 1
+}
+
+// Pauli X
+static L_X_OUT0: [LocalEntry; 1] = [LocalEntry(1, Complex64::new(1.0, 0.0))];
+static L_X_OUT1: [LocalEntry; 1] = [LocalEntry(0, Complex64::new(1.0, 0.0))];
+
+// Pauli Z
+static L_Z_OUT0: [LocalEntry; 1] = [LocalEntry(0, Complex64::new(1.0, 0.0))];
+static L_Z_OUT1: [LocalEntry; 1] = [LocalEntry(1, Complex64::new(-1.0, 0.0))];
+
+// Pauli Y
+static L_Y_OUT0: [LocalEntry; 1] = [LocalEntry(1, Complex64::new(0.0, -1.0))]; // -i
+static L_Y_OUT1: [LocalEntry; 1] = [LocalEntry(0, Complex64::new(0.0, 1.0))]; // +i
```

Comment:
The use of static is very good, but there are some duplications here:
L_X_OUT0 is the same as L_ONE_OUT1, 
L_X_OUT1 is the same as L_Z_OUT0 and L_ZERO_OUT0,
etc.

Is there a simpler way to define them only once?
See e.g. 
https://github.com/Qiskit/qiskit/blob/main/crates/circuit/src/util.rs
https://github.com/Qiskit/qiskit/blob/main/crates/circuit/src/gate_matrix.rs